src/Content/ContentStorage.php

```diff
@@ -217,6 +217,18 @@ public function read(
 		return $this->handler->read($version, $lang);
 	}
 
+	/**
+	 * Searches and replaces one or multiple strings in all versions
+	 *
+	 * @param array $map search-replace mapping where all keys are replaced by their values
+	 */
+	public function replace(array $map): void
+	{
+		foreach ($this->all() as $version => $lang) {
+			$this->handler->replace($version, $lang, $map);
+		}
+	}
+
```

Comment:
The ContentStorage class no longer exists in the alpha. Is there any chance that we can implement this without relying on the wrapper class? I guess we will otherwise create some nasty merge conflicts later.

Reply:
Maybe just adding a comment? Cause I am not sure if trying to work around it wouldn't make quite the mess for 4.5 – or we directly target v5 with this from the start?

Reply:
Seeing all the work ahead of us for the changes, I would probably prefer to put it in v5 and not cause additional conflicts with it tbh.